airbyte-webapp/src/pages/destination/AllDestinationsPage/index.tsx

```diff
@@ -1 +1,4 @@
+import { AllDestinationsPage } from "./AllDestinationsPage";
+
 export { AllDestinationsPage } from "./AllDestinationsPage";
+export default AllDestinationsPage;
```

Comment:
Could we just do the default export here? I don't think we need the named one at all.